elastic/kibana
# [Custom threshold] Add group by filters to the custom threshold view in app URL

Resolves #173713

## Summary

This PR adds group by filters to the view in the app URL for the custom threshold rule:

![image](https://github.com/elastic/kibana/assets/12370520/c84eecfc-b196-4268-acdd-57bae86ac4c4)

I also moved some types to the common folder and adjusted them to match the reality.

## 🧪 How to test
- Create a custom threshold rule with group by
   - One with persisted data view
   - One with an ad-hoc data view
- Check the view in app link in the alerts table, you should also see the group filters there.
- Check the view in app URL from the actions, it should also include the group filters.
